modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm

```diff
@@ -55,6 +55,17 @@
 	lizard.update_mutant_bodyparts(TRUE)
 	lizard.update_body(TRUE)
 
+/datum/species/lizard/get_species_description()
+	return {"Lizardperson is a catch-all term for any anthropomorphic creature, often reptilian, \
+	that is either not registered in any official Sapient Universal Species Database under the Sol Federation, Taj Empire, Agurrkral Kingdom, Novaya Rossiyskaya Imperiya, and other major States \
+	- or is simply a result of genetical modification."}
```

Comment:
```suggestion
	- or is simply a result of genetic modification."}
```
While Genetical is TECHNICALLY a word its so horribly gross

Reply:
Yes I forgot about this